### PR TITLE
Erik Readme and run.bat updates

### PR DESCRIPTION
**Pull request by eneemann**
Some minor updates to the readme and a fix for the run.bat environment variable.

### CODE REVIEW

File: src/masquerade/providers/web_api.py [hunk 1]
```diff
@@ -52,10 +52,10 @@ def etl_candidate(agrc_candidate):
     return {
         'address': address,
         'attributes': {
-            'score': agrc_candidate['score'],
+            'Score': agrc_candidate['score'],
             'locator': agrc_candidate['locator'],
             'addressGrid': agrc_candidate['addressGrid']
```
Comment (stdavis) [inline src/masquerade/providers/web_api.py:57]: **question**: I wonder if `locator` and `addressGrid` should be title-cased as well just for consistency? Same question for `address`, `attributes`, & `location` in the parent object. What does the Esri locator service return object look like?

**Review by stdavis — CHANGES_REQUESTED** [review]
Looks like the casing change broke some tests. I would suggest that you figure out the rest of the casing (see my in-line comment) and then I can show you how to run the tests and fix them.

File: src/masquerade/providers/web_api.py (hunk 1 above)
Comment (eneemann) [reply src/masquerade/providers/web_api.py:57]: I'll try to take a look later this week to see what I can figure out.

**Review by stdavis — APPROVED** [review]

**Comment by stdavis** [comment]
@eneemann These are really helpful commits! Thanks!

**Pull request merged**